WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -106,12 +109,13 @@ public async Task<bool> StartRoundAsync(IEnumerable<SmartCoin> smartCoins, Round
 				var registeredCoins = registeredAliceClients.Select(x => x.SmartCoin.Coin);
 				var availableVsize = registeredAliceClients.SelectMany(x => x.IssuedVsizeCredentials).Sum(x => x.Value);
 
-				// Calculate outputs values
+				var amountDecomposer = await amountDecomposerTask;
```

Comment:
```suggestion
				var amountDecomposer = await amountDecomposerTask.ConfigureAwait(false);
```

Reply:
Saw this suggestion in VS and I think it makes sense based on what I read here: https://medium.com/bynder-tech/c-why-you-should-use-configureawait-false-in-your-library-code-d7837dce3d7f

Reply:
https://github.com/zkSNACKs/WalletWasabi/blob/master/CONTRIBUTING.md#configureawaitfalse